build.sbt

```diff
@@ -5,16 +5,17 @@ lazy val scalaSettings = Seq(
   scalacOptions += "-feature",
   crossScalaVersions := Seq("2.12.8"),
   libraryDependencies ++= Seq(
-    "org.scalatest" %% "scalatest" % "3.0.8" % "test"
+    "org.scalatest" %% "scalatest"               % "3.0.8"  % "test",
```

Comment:
why not `% Test`? Also, isn't it best practice to put all dependencies into `Dependencies.scala` file?